action.yml

```diff
@@ -76,18 +79,27 @@ outputs:
       }
       ```
     value: ${{ steps.update_files.outputs.json }}
+  changed_files:
+    description: |
+      A newline-separated list of files that were modified during the digest update process.
+      Only includes files that actually had their digests updated.
+    value: ${{ steps.update_files.outputs.changed_files }}
 
 runs:
   using: "composite"
   steps:
     - uses: imjasonh/setup-crane@31b88efe9de28ae0ffa220711af4b60be9435f6e # v0.4
-
+    - uses: nais/login@e7cf2c159677dc7c7d599feff5f808f2bf59c7cf # v0
```

Comment:
Why is this needed?

Reply:
this should be added in the user workflow not here

Reply:
That was not supposed to be here. 

But it gives Nav developers access to our GAR repository where we have a chainguard pull-through.